frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -839,13 +848,19 @@ export class Query extends Component {
                             }
                           />
                           <Popup
-                            content="Add User"
+                            content=<a href="https://github.com/wikimedia/WikiContrib"
+                              rel="noopener noreferrer" target="_blank">
+                              Contact The maintainers of this Project to learn more about this feature
```

Comment:
Change the message to "This feature is currently disabled. Contact the tool maintainers to learn more."